src/sagemaker/hyperpod/cli/recipe_utils/recipe_loader.py

```diff
@@ -1,62 +1,109 @@
 import os
 import yaml
-from typing import Dict, Any, Optional
+from hyperpod_pytorch_job_template.v1_0.recipes.hf.model import HfRecipeSchema
```

Comment:
Here we are directly loading from v1_0 but what we need is to use the version aware registry pull up? 
is this something we will iterate on based on recent change from Roja?